src/api/RestJobbprofil.ts

```diff
@@ -0,0 +1,40 @@
+export type RestJobbprofil = Suksess | IkkeLastet | LasterInn | IkkeFunnet | Feil | UkjentFeil;
+
+export enum Status {
```

Comment:
Tok med litt flere statuser enn vi bruker foreløpig, samme som er for hent kandidat.